stdlib/public/runtime/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ list(APPEND swift_runtime_compile_flags
      "-D__SWIFT_CURRENT_DYLIB=swiftCore")
 
 # SWIFT_ENABLE_TENSORFLOW
+find_package(TensorFlow REQUIRED)
```

Comment:
What does this do?

Reply:
Update: I moved code in the new files TensorFlow.{h,cpp} to the CTensorFlow
 module (ctensorflow_init.{h,cpp}), so that the stdlib/public/runtime/ need not
 depend on the TF library. As such I reverted the changes to the CMakefile here.

More context for the original plan:
`find_package()` uses the swift/cmake/modules/FindTensorFlow.cmake script to define the TF_INCLUDE_DIR variable, which the next line uses, so that the new runtime source file swift/stdlib/public/runtime/TensorFlow.cpp can find header files like "tensorflow/c/c_api.h".

https://cmake.org/cmake/help/v3.0/command/find_package.html has more context.